src/encoded/static/components/dataset.js

```diff
@@ -121,6 +121,27 @@ const AnnotationComponent = (props, reactContext) => {
                                 </div>
                             : null}
 
+                            {context.assay_term_name ?
+                                <div data-test="assaytermname">
+                                    <dt>Assay</dt>
+                                    <dd>{context.assay_term_name}</dd>
+                                </div>
+                            : null}
+
+                            {context.targets && context.targets.length > 0 ?
+                                <div data-test="targets">
+                                    <dt>Target</dt>
+                                    <dd>
+                                        {context.targets.map((target, i) =>
+                                        <span>
+                                            {i > 0 ? <span>, </span> : null}
+                                            <a href={target['@id']}>{target.label}</a>
+                                        </span>
+                                        )}
```

Comment:
You’ll need to indent this `<span>` section one to the right. It shows up as a formatting error in my editor.

Also, and optionally, you can replace the `<span>`/`</span>` tags (the ones within the `context.target` loop; not the ones around the comma) with `<React.Fragment>`/`</React.Fragment>` tags. The advantage is the generated HTML with React.Fragment won’t include `<span>` tags; it generates no tags at all so the HTML is a little more efficient.